### ISSUE TITLE
error when stopping execution before execution has started

### ISSUE THREAD

**Issue by gordonwoodhull** [att/rcloud#1441]
As reported by @sanketd11, it's possible to interrupt cell execution before it has actually started, producing a harmless but annoying error:

![image006](https://cloud.githubusercontent.com/assets/1366709/7597944/0437abc4-f8c5-11e4-8ed1-a584815cf09c.jpg)

Presumably other system compute-process code can also be accidentally interrupted. The same error can also appear on other lines of code.

![image](https://cloud.githubusercontent.com/assets/1366709/7598228/eaed9a64-f8c6-11e4-9626-618f2dea1e7f.png)

**Comment by gordonwoodhull** [comment]
should log a polite message to session pane instead.

**Comment by gordonwoodhull** [comment]
@s-u, this is waiting on the context going first in OOB sends/messages, to handle contexts in a consistent way.

**Comment by gordonwoodhull** [comment]
@s-u, actually I'm not sure what to do here. Surely we don't want yet another try-catch? Many different places can fail here.

I know the expected behavior is just to return a regular `cell-eval-error` as if evaluat***ion*** (not evaluat***ing***) failed

**Comment by gordonwoodhull** [comment]
passing this back, R guru!

**Comment by gordonwoodhull** [comment]
unless there's really no good way to deal with this, since there are legitimate failures like unable to fetch the cell content.

**Comment by s-u** [comment]
But those are already covered by the context, so I'm not sure what exactly is the issue. You will always be able to interrupt before we can even setup a context - that is absolutely expected by definition - it's an asynchronous even after all. So is this really about #1473? Interrupts shouldn't really go to the session pane regardless of where they happen...

**Comment by gordonwoodhull** [comment]
Maybe we don't have to deal with it for 1.4, but it seems pretty weird that pressing the stop button will generate a big scary error message depending on the moment you press it.

**Comment by s-u** [comment]
Well, it is #1473 and I'm sorry to say that we probably won't be able to solve it without hacking R. The `tryCatch(..., interrupt=..)` approach doesn't seem to work with those nested evals we're dealing with. That said, we should be (hopefully) able to avoid it for the most common case which seems to be interrupting the retrieval of a long notebook.
